Objects/Elf/Elf.gd

```diff
@@ -12,7 +12,7 @@ var next_arrow_velocity : Vector2
 
 var hp : float setget set_current_hp
 
-onready var stamina_system = get_parent().find_node("StaminaLab")
+onready var stamina_system = get_parent().get_parent().find_node("StaminaLab")
```

Comment:
It work? stamina_system is not empty when you remove one get_parent() ? if yes OK

Reply:
I  can't remove one get_parent(), because Elf is in CanvasLayer (since I added parallax background and she had to stay in one place), so it isn't direct child of World node.